docs/diagrams/UndoRedoState1.puml

```diff
@@ -6,9 +6,9 @@ skinparam ClassBorderColor #000000
 title After command "delete 5"
 
 package States <<rectangle>> {
-    class State1 as "__ab0:AddressBook__"
-    class State2 as "__ab1:AddressBook__"
-    class State3 as "__ab2:AddressBook__"
+    class State1 as "__ab0:MediBook__"
+    class State2 as "__ab1:MediBook__"
+    class State3 as "__ab2:MediBook__"
```

Comment:
Same as for UndoRedoState0.puml, maybe we should change in all the diagram files?